README.md

```diff
@@ -115,6 +116,11 @@ versions for the best experience. These can be found
     `defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false`.
 -   To fix the remapped escape key not working in Linux, set `"keyboard.dispatch": "keyCode"`
 
+### Settings & Commands
+
+Please open the [vscode-neovim](vscode:extension/asvetliakov.vscode-neovim) extension details in VSCode, and check all
+available [settings](https://code.visualstudio.com/docs/getstarted/settings) and commands in the Features tab.
```

Comment:
I don't think this link is necessary

Reply:
Which one? (Please do not trust your user.) I referred to the documentation of other extensions, such as vscode-eslint.

Reply:
The one to the official docs. Given the number of places we refer to various settings in this README, it seems strange to presume they don't know VSCode can be configured

Reply:
:joy: